modules/system/database/migrations/2021_06_25_143510_Db_Attachment_ID_To_int.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+use Illuminate\Support\Facades\DB;
+use Winter\Storm\Database\Schema\Blueprint;
+use Winter\Storm\Database\Updates\Migration;
+use Winter\Storm\Support\Facades\Schema;
+
+class DbAttachmentIdToInt extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('system_files', function (Blueprint $table) {
+            // Drop old indexes
+            $table->dropIndex(['attachment_id']);
+            $table->dropIndex(['attachment_type']);
+            $table->dropIndex(['field']);
```

Comment:
bracket is unnecessary here

Reply:
Supplying the index name as an array actually allows us to just specify the index to drop by field name instead of needing to include the prefix and suffix that are automatically generated by Laravel when creating the index. Basically, it just allows us to omit the `system_files_` prefix and the `_index` suffix from the index name, because the full index name is actually `system_files_field_index`

Reference: https://github.com/laravel/framework/blob/6.x/src/Illuminate/Database/Schema/Blueprint.php#L1319